jsonnet/main.jsonnet

```diff
@@ -320,7 +320,30 @@ local inCluster =
     },
 
     // Objects
-    clusterMonitoringOperator: clusterMonitoringOperator($.values.clusterMonitoringOperator),
+    clusterMonitoringOperator: clusterMonitoringOperator($.values.clusterMonitoringOperator) {
+      // The cluster-monitoring-operator ClusterRole needs the combined set of
+      // permissions from all its operand ClusterRoles.  This extends the base
+      // ClusterRole by just appending the rules from the others.
+      clusterRole+: {
+        rules+: inCluster.alertmanager.clusterRole.rules +
```

Comment:
nit: I just remembered there is this thing called [Sets in jsonnet stdlib](https://jsonnet.org/ref/stdlib.html). It would be nice to use it here. Just wrap all this in `std.set([ .... ])` and we should have a nicely sorted and partially deduplicated list of rules :)

Reply:
Ah, cool. Done.